Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -136,15 +136,16 @@ tasks:
       format:
         simple: xml
       sha256:
-        simple: ${inputs.FileSha256}
+        complex:
+          root: inputs.FileSha256
```

Comment:
Please add a unique transformer - avoiding duplicates.

Reply:
done - added to tasks number 6 and 5.